source/frontend/src/components/common/HeaderField/HeaderField.tsx

```diff
@@ -33,7 +35,7 @@ export const HeaderField: React.FC<React.PropsWithChildren<IHeaderFieldProps>> =
 };
 
 export const HeaderLabelCol: React.FC<IHeaderLabelColProps> = props => (
-  <Col xs={props.labelWidth ?? 'auto'} className="pr-0 text-left">
```

Comment:
Prior to this all our header labels/content were forced to use xs, however, there are a number of instances in the application where that isn't sufficient.